semgrep/semgrep/error.py

```diff
@@ -1,20 +1,24 @@
-from typing import Optional
-
-
-class SemgrepException(Exception):
+class SemgrepError(Exception):
     """
     Parent class of all exceptions we anticipate in Semgrep commands
 
     All Semgrep Exceptions are caught and their error messages
     are displayed to the user.
     """
 
-    def __init__(self) -> None:
-        self.msg: Optional[str] = None
```

Comment:
There's no need to smuggle an additional `msg` attribute, we can just call `str(e)` to get the message.